### DISCUSSION TITLE
Make Build Using Visual Studio 2022

### DISCUSSION THREAD

**Discussion by HHT-TH** [ttadano/alamode#224] [Q&A]
I am interested in theoretical calculations that take into account temperature and want to use alamode.
However, when I followed the instructions in the documentation and executed cmake -DUSE_MKL_FFT=no .., an error occurred. The error message indicated that this was due to the absence of Visual Studio 2022. So, I installed Visual Studio 2022 and successfully executed cmake -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DUSE_MKL_FFT=no ...
However, when I tried to build the project within Visual Studio 2022, more than 1000 errors occurred, and I was unable to complete the build.
Due to my lack of knowledge, I apologize for the inconvenience, but could you please guide me on how to correctly install and build ALAMODE?
Below are the details of my computer and the software I have installed:
Windows11
Anaconda
OneAPI base Toolkit
OneAPI HPC Toolkit
Visual Studio 2022

**Comment by ttadano** [comment]
I remember that I could compile the code using the Visual Studio Compiler long time ago, but I have not tested it for a while. Perhaps, some libraries or compiler options are missing.

Currently, I do not have environment to test the Visual Studio Compiler, so I suggest that you use a compiler from conda.
